processors/processor_text_whisper.py

```diff
@@ -0,0 +1,51 @@
+from processor_base import BaseProcessor
+import whisper
+import math
+
+@BaseProcessor.register_processor('text_whisper')
+class TextWhisperProcessor(BaseProcessor):
+
+    def __init__(self, *args, **kwargs):
+        self.init_processor()
+
+    def init_processor(self):
+        self.model_id: str = 'base'
+
+    def process(self, payload: dict) -> dict:
+        input = payload["inputs"]
+        model = whisper.load_model(self.model_id)
```

Comment:
Hi Aprille, it should not do whisper.load_model. It indicates that the model is running on the local machine instead of hosting on another machine and call it via API.